docs/wiki/deployment/aws-logging.md

```diff
@@ -11,6 +11,10 @@ Some configuration is shared between the two plugins:
 --aws_profile_name VALUE                AWS config profile to use for auth and region config
 --aws_region VALUE                      AWS region override
 --aws_secret_access_key VALUE           AWS secret access key override
+--aws_sts_arn_role VALUE                AWS STS assume role arn
+--aws_sts_region VALUE                  AWS STS assume role region
```

Comment:
Unfortunately, its the case for me most of the time.  Ill test it tomorrow to see if it works without specifying an assume role region but I think I have tried that before.